### DISCUSSION TITLE
GitHub Profile Readme

### DISCUSSION THREAD

**Discussion by EmekaDollar** [community/community#49891] [Repositories]
### Select Topic Area

General

### Body

Please, I have been unable to display my special readme repository on my public GitHub profile. When I click on the “share to profile” button, it redirects me to a regret page. Please help. 
![C16B3FDF-3C0A-4A4C-B91B-36FF72FD0D33](https://user-images.githubusercontent.com/100249337/224620684-c6deb7d1-ae72-4d30-bf80-17faef473849.png)

**Comment by bkpecho** [comment]
Hello brother @EmekaDollar 

I see you created a beautiful README.md profile but it's not showing.

Please try these steps. I hope it fixes your issue.

1. Check your repository settings to make sure that the "Include in profile README" option is enabled. To do this, go to your repository's Settings > Options > Features and make sure that the "Included in profile README" option is checked.
2. Clear your browser cache and try again. Sometimes browser cache can cause issues with GitHub's functionality.
3. Try accessing the "share to profile" button from a different browser or device to see if the issue persists.

**Comment by EmekaDollar** [comment]
I have tried all this options, it’s still not working. I’m frustrated.
>
